pydocx/openxml/markup_compatibility/fallback.py

```diff
@@ -6,10 +6,8 @@
 )
 
 from pydocx.models import XmlModel, XmlCollection
-from pydocx.openxml.wordprocessing.picture import Picture
 
 
 class Fallback(XmlModel):
     XML_TAG = 'Fallback'
-    # TODO #204: actually include all of the children defined in the spec.
-    children = XmlCollection(Picture)
+    children = XmlCollection(allow_all_children=True)
```

Comment:
OK. I just also spent a half hour. I see what we need to do, but it's somewhat complicated. Can you add a comment here explaining that it would be better to refer to the grandparent's `children` XmlCollection lazily? The problem is that we don't have a good way to represent lazy fields, nor do we have a way for those fields to reference their parents before they are initialized with content.